src/Searcher.tsx

```diff
@@ -6,18 +6,11 @@ import { LibraryContainer } from "./components/LibraryContainer";
 import * as LibraryUtilities from "./LibraryUtilities";
 import { SearchBar } from "./components/SearchBar";
 
-type displayMode = "structure" | "list";
-
-interface SearchModeChangedFunc {
-    (inSearchMode: boolean): void;
-}
-
```

Comment:
I am not sure why this is added for `Searcher`, as it is not used at all...so I just removed it